dapp/src/app/actions/colony.ts

```diff
@@ -0,0 +1,54 @@
+const { providers, Wallet } = require("ethers");
+const { default: EthersAdapter } = require("@colony/colony-js-adapter-ethers");
+const { TrufflepigLoader } = require("@colony/colony-js-contract-loader-http");
+
+const { default: ColonyNetworkClient } = require("@colony/colony-js-client");
+const loader = new TrufflepigLoader();
```

Comment:
Trufflepig is only for development purposes. We should try to keep this dependency out of our product code.

I'm not sure what would go in it's place though, I'd have to dig in more.

Reply:
Just did some reading, here's how I understand it:
- A a contract deployed to the blockchain has two things, an address and an ABI (Application Binary Interface).
- The wallet needs the ABI in order to know how to form the transaction for specific methods (member functions) it want so to call on the contract.
- The loader's job is to return that ABI.
- TrufflePig is a handy loader for local development. It assumes all contracts have a unique name, so you can ask for an ABI by saying "getContract('contractName')".
- For production, when the contracts are on the network (mainnet or testnet), the loader could ask etherscan.io for the ABI of a specific public contract address.

Here's more information: 
https://docs.colony.io/colonyjs/docs-loaders/ 

I'll create a task for making sure we have this implemented correctly for both local development and production.

Reply:
https://github.com/JingyuZ/GithubColony/issues/17